assets/js/components/dashboard/EventField.tsx

```diff
@@ -38,7 +45,7 @@ const EventField = ({ webinarInfo }: EventFieldProps) => {
             <CalendarClock />
             {formatDate(webinarInfo.startDate)}
           </div>
-          {isAuthenticated && (
+          {isAuthenticated && userEmail == webinarInfo.moderatorEmail && (
```

Comment:
Maybe this would be a little clearer?

```suggestion
          const isUserEventModerator = userEmail == webinarInfo.moderatorEmail;
          (...)
          {isAuthenticated &&  && (
```